### PR TITLE
add ENULS network

### PR DESCRIPTION
**Pull request by MIMIEYES**
add ENULS network

### CODE REVIEW

File: src/networks.json [hunk 1]
```diff
@@ -2118,5 +2118,20 @@
...
     "logo": "ipfs://QmYGxmEhV6djksUk97pdE9TmL6DXm9KW8BP7pwUv8pqp8r"
+  },
+  "119": {
+    "key": "119",
+    "name": "ENULS Mainnet",
+    "chainId": 119,
+    "network": "mainnet",
+    "multicall": "0xd7d2Aa52f5491cB60ccD85b8c39935BF0baCd142",
+    "rpc": [
+      "https://evmapi.nuls.io/"
```
Comment (ChaituVR) [inline src/networks.json:2129]: Hi @MIMIEYES this is not a archive node, it return error `missing trie node`

File: src/networks.json [hunk 2]
```diff
@@ -2118,5 +2118,20 @@
...
+    ],
+    "explorer": {
+      "url": "https://evmscan.nuls.io"
+    },
+    "start": 897419,
```
Comment (ChaituVR) [inline src/networks.json:2134]:
```suggestion
    "start": 501318,
```

**Review by ChaituVR — COMMENTED** [review]
everything else looks good

**Review by ChaituVR — CHANGES_REQUESTED** [review]
everything else looks good

File: src/networks.json (hunk 1 above)
Comment (MIMIEYES) [reply src/networks.json:2129]: > 

Hi @ChaituVR we have changed it to an archive node, pls check.

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**